### ISSUE TITLE
WebExtensions settings

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#6091]
### User Story
As a user, I want to be able to access the settings for my installed WebExtensions, so I can customize my settings to match my preferences. 

### Dependencies
[A-C project board](https://github.com/orgs/mozilla-mobile/projects/44)

### Acceptance Criteria
- I can access and modify the settings for my installed WebExtensions
- I can view details (description of the WebExtension, author, version, last updated, etc)
- I can view the list of permissions I have granted to my WebExtensions
- I can choose to allow an installed WebExtension in private browsing

### Design tasks
- Provide a way to access Settings for installed WebExtensions
    - [ ] Within Fenix Settings
    - [ ] Within the main interface of each installed WebExtensions

**Comment by brampitoyo** [comment]
Added design tasks.

**Comment by brampitoyo** [comment]
Progress report:

* I’ve added a way to access uBlock Origin Settings
    * uBlock Origin is not optimised for mobile, so the UI will be hard to use
    * **We need to talk to the developer to help adapt their UI for mobile layout**
* I’ve added a way to navigate to/from uBlock Origin Settings from within the Add-ons Manager
    * This seems to be all good and fairly straightforward.
* When solved for uBlock Origin, we will also solve this problem for all other WebExtensions

**Comment by brampitoyo** [comment]
Progress report:
* We’ve confirmed that auto-update is not modifiable. It’s always set to ON. This means that all add-ons will auto-update to the newest version that’s recommended by AMO.
* We’ve also confirmed that all add-ons will run in both private and normal browsing
    * In the future, we may implement a control that make it possible for a specific add-on to turn OFF while in private browsing.

**Comment by softvision-miralobontiu** [comment]
Verified that, as a user, I can:
- access and modify the settings for uBlock Origin,
- view details and description of the uBlock Origin's author, version, last updated, etc., 
- view the list of permissions I have granted to uBlock Origin,
- observe uBlock Origin run in both private and normal browsing.

Tested on the latest Nightly 200205 (#20360607) with One Plus 5T (Android 9), Google Pixel (Android 10), Samsung Galaxy Tab S3 (Android 8).

I will remove the eng:qa:needed label.
@brampitoyo can I close this?
